<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.automation.runners.loginRunner2" tests="1" time="6.073" ignored="0" failures="0" timestamp="07 Jul 2020 13:14:17 GMT" skipped="0" hostname="LAPTOP-AV2ODUL9" errors="1">
  <testcase name="Login" time="6.073" classname="com.automation.runners.loginRunner2">
    <error type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: Actuals did not match with expected:: expected [20] but found [200]">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: Actuals did not match with expected:: expected [20] but found [200]
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at com.automation.runners.loginRunner2.Login(loginRunner2.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: Actuals did not match with expected:: expected [20] but found [200]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at com.automation.stepdefinitions.StepDefination.the_API_call_got_success_with_status_code(StepDefination.java:60)
	at âœ½.Then the API call got success with status code 200(src/test/java/com/automation/features/placeValidations.feature:5)
]]>
    </error>
  </testcase> <!-- Login -->
</testsuite> <!-- com.automation.runners.loginRunner2 -->
